src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -474,12 +474,36 @@ public void InsertCode(string text)
                 }
                 else
                 {
-                    if (!TextView.Selection.IsEmpty)
+                    using (var transaction = UndoHistory_TestOnly.CreateTransaction(InteractiveWindowResources.TypeChar))
```

Comment:
@dpoeschl the `_TestOnly` suffix is to indicate this property is exposed only for testing and shouldn't be used outside of this class for any other purposes. Should I just change it and use comments instead?